### ISSUE TITLE
Negative probabilities in model output

### ISSUE THREAD

**Issue by bobogumizelka** [jayrodge/Binary-Image-Classifier-PyTorch#1]
I printed output probabilities during inference, and they're negative. How is that possible? There is a softmax in the model...
